.github/workflows/debug_build.yml

```diff
@@ -15,9 +15,6 @@ jobs:
       - name: Install Protoc
         uses: arduino/setup-protoc@v2
 
-      - name: Install capnproto
```

Comment:
I don't see any changes related to this. Why don't we need capnproto anymore?

Reply:
It was needed transitively for the CDN, but now the generated code is pinned